Userland/Utilities/ping.cpp

```diff
@@ -26,7 +26,7 @@
 
 static uint32_t total_pings;
 static int successful_pings;
-static uint32_t count;
+static uint32_t count = -1;
```

Comment:
This will just underflow `count`, shouldn't the default be `0`? 
Which then means infinite packets?
Maybe that should also be mentioned in the argument text. Something like:
`"Stop after sending specified number of ECHO_REQUEST packets, use 0 to never stop (default: 0)."`

Reply:
Why don't we store `count` as `Optional<size_t>`? Might need to add support for that to `ArgParser`, but then we can use `count`'s empty state instead of a special `-1` value here.

Reply:
Fair enough! I can look into it :+1:

Reply:
@trflynn89 yeah, this feels a lot better. Let me know what you think